app/src/main/res/layout/activity_splash.xml

```diff
@@ -26,6 +26,18 @@
         android:textSize="@dimen/splash_subtitle_textsize"
         />
 
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/ongoing_contests_recycler_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_above="@+id/splash_screen_actions_layout"
+        android:layout_alignBottom="@+id/splash_screen_actions_layout"
+        android:layout_below="@id/prompt_sub_header"
+        android:layout_marginLeft="@dimen/activity_horizontal_margin"
```

Comment:
Let's try to stick to marginstart/end instead of left/right. Its a good habit to get into.